specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/blob.json

```diff
@@ -797,6 +797,14 @@
   "definitions": {
     "ContainerProperties": {
       "properties": {
+        "defaultEncryptionScope": {
+          "type": "string",
+          "description": "Default the container to use specified encryption scope for all writes."
+        },
+        "denyEncryptionScopeOverride": {
```

Comment:
>denyEncryptionScopeOverride [](start = 9, length = 27)

Consider to to use enum instead of boolean in case you need other mode which may cause breaking change.

Reply:
This was intentionally modeled as a boolean to match data plane headers following the same.